fastlane/Fastfile

```diff
@@ -146,7 +182,7 @@ platform :ios do
   desc 'Creates a new release branch from the current trunk'
   lane :complete_code_freeze do |options|
     ios_completecodefreeze_prechecks(options)
-    update_localization
+    generate_strings_file_for_glotpress
```

Comment:
I took the chance to update this to what seems to me like a clearer name, which is the same as [what WordPress uses](https://github.com/wordpress-mobile/WordPress-iOS/blob/b8e95b1d57d6b3a3a4ab7501199b0216eadc3d03/fastlane/lanes/release.rb#L64)